pkgs/by-name/ee/eepers/package.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gnat
+, raylib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "eepers";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "tsoding";
+    repo = "eepers";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-KG7ci327qlTtlN4yV54P8Q34ExFLJfTGMTZxN3RtZbc=";
+  };
+
+  buildInputs = [
+    gnat
+    raylib
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    gnatmake -f -O3 \
+      -Wall \
+      -Wextra \
+      -gnat2012 \
+      -o eepers-linux eepers.adb \
+      -bargs \
+      -largs -lraylib -lm \
+      -pthread
+
+    runHook postBuild
+  '';
```

Comment:
maybe pass `buildFlags` instead of setting the `buildPhase`?

Reply:
I just tried and it doesn't work. Looks like adding `gnat` to the `nativeBuildInputs` is not enough to generate a build phase.
It seems like explicitly writting the `buildPhase` is necessary...

Reply:
Well that resolved in this case